### PR TITLE
Adding the qgrid recipe

### PR DESCRIPTION
**Pull request by TimShawver**
It’s basically a copy of the widgetsnbextension recipe with some slight changes to make it qgrid-specific. Here's that recipe's latest pull request, for reference: https://github.com/conda-forge/staged-recipes/pull/3655

### CODE REVIEW

File: recipes/qgrid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  version: {{ version }}
+
+source:
+  fn: qgrid-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/q/qgrid/qgrid-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (jakirkham) [inline recipes/qgrid/meta.yaml]: Minor comment: Would use 2 space indenting here as it was used elsewhere in the recipe.

File: recipes/qgrid/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - ipywidgets >=7.0.0
+
+test:
+  imports:
+    - qgrid
+
+about:
+  home: https://github.com/quantopian/qgrid
+  license: Apache 2.0
+  license_family: Apache
```
Comment (jakirkham) [inline recipes/qgrid/meta.yaml]: Please add `license_file: LICENSE`.

**Comment by TimShawver** [comment]
@jakirkham I made those changes but it looks like tests are failing now, any idea why?

**Comment by jakirkham** [comment]
Sorry kind of busy these days. Try merging with `master` to fix the CircleCI issue.

**Comment by TimShawver** [comment]
@jakirkham thanks it looks like that fixed the CircleCI issue but now the Travic CI build appears to be hanging (it never finishes).

**Comment by TimShawver** [comment]
@jakirkham anything else you need from me before merging this? As I mentioned the Travis CI build never finished...not sure if that's a problem on my end though.

**Comment by minrk** [comment]
Travis came back, everything looks OK now

**Pull request merged**